Makefile

```diff
@@ -16,6 +16,11 @@ RIOTER_FOOTER_PACKAGE=rioter-footer-nft
 VAULT_REPO=teritori-vault
 VAULT_PACKAGE=teritori-nft-vault
 
+DAO_CONTRACTS_REPO=dao-contracts
```

Comment:
this should not use this repo but instead use upstream DA0-DA0

Reply:
You right, fixed !